packages/loader/container-loader/src/containerContext.ts

```diff
@@ -97,9 +96,7 @@ export class ContainerContext implements IContainerContext {
 
     /** @deprecated Added back to unblock 0.56 integration */
     public get id(): string {
-        const resolvedUrl = this.container.resolvedUrl;
-        ensureFluidResolvedUrl(resolvedUrl);
```

Comment:
@chensixx Is there a reason `ensureFluidResolvedUrl` was added here? This was not in the original code before it was removed.